### PR TITLE
Added Enemy Class

### PR DESCRIPTION
**Pull request by JLeezy93**
I added Enemy Entity,

### CODE REVIEW

**Comment by mmonfils** [comment]
Issue #27

File: Master-Game/Entities/Enemy.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace MasterGame.Entities
+{
+    public class Enemy : HumanEntity
```
Comment (jamiahuswalton) [inline Master-Game/Entities/Enemy.cs]: Enemy >> EnemyEntity

File: Master-Game/Entities/Enemy.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    {
+        public Enemy()
+            : base ("Evil Guy", 500)
```
Comment (jamiahuswalton) [inline Master-Game/Entities/Enemy.cs]: Will need to indicate max health?

Comment (jamiahuswalton) [inline Master-Game/Entities/Enemy.cs]: Will need to allow people to indicate their name and health. As it stands, every time an enemy is created, the name and health will always be "Evil Guy" and 500, respectively.

File: Master-Game/Entities/Enemy.cs (hunk 1 above)
Comment (jamiahuswalton) [inline Master-Game/Entities/Enemy.cs]: When this is created, the base is a human entity. This means that the enemy will be a human type. Is this something that we want? Or do we want to create an enemy entity?

**Comment by havok13888** [comment]
The enemy entity is too generic. We will make specific ones at a later period

**Pull request closed without merging**